### ISSUE TITLE
If `xml:lang` then direction is also needed...

### ISSUE THREAD

**Issue by iherman** [w3c/sync-media-pub#54]
(Looking ahead of a possible i18n review...)

The document mentions the usage of `xml:lang`. As far as I can see, the only element that is affected by this is `metadata`; according to the content model that is the only element which, potentially, may contain human-readable text. However, if the language is set, then we also have to have the possibility to set the base direction, ie, a `sync:dir` attributes becomes necessary (copying, e.g., the EPUB attribute).

I see these possibilities:

1. We leave the spec as is, and introduce `sync:dir`
2. We do _not_ mention `xml:lang` in the content model, and we say (e.g., in a note) that the various metadata from "any metainformation structuring language" are supposed to define ways of setting language and base direction and this specification does not deal with the problem.

**Comment by iherman** [comment]
Note that SMIL pretty much ignores the language and base directionality issue (unless I missed it); alas!, there wasn't such a strong i18n review practice in W3C back then.

**Comment by marisademeglio** [comment]
You're right that we have mistakenly left out `dir`. As for where language and dir could be applicable, I could see the document itself having a language declaration -- this could go in `metadata` like you wrote. 

However, the `sync:label` attribute (which provides a label for a track) is human-readable text and could have a language + direction. And to express that properly, we should probably allow many `sync:label`s, whereas right now we only have one .. and then we should probably make it an element and not an attribute. 

Or, we choose your (2) above and also drop `sync:label`, and leave it up to the user agents to provide labels for tracks, based on our (TBD) list of common track types. E.g.

```
<sync:track sync:trackType="soundfx" ... />
```

where `soundfx` comes from our registered track types list (see [your comment](https://github.com/w3c/sync-media-pub/issues/31#issuecomment-749406336)). 

And that should be enough information for a user agent to label the track itself, and use its own localization settings.

**Comment by iherman** [comment]
Ah! I missed `sync:label`. Yes, if we keep it then this means we definitely need `sync:dir`.

Whether we need it or not: I cannot really comment on that; I miss the necessary usage experience.
